app/decorators/post_decorator.rb

```diff
@@ -8,7 +8,7 @@ def to_og_params
       locale: 'ja_JP',
       type: 'article',
       title: title,
-      description: render_markdown(body),
+      description: render_markdown(body)&.gsub("\n", ''),
```

Comment:
By the way, posts always should have some contents.
So how about adding not null constraint to `posts.body`?